schema.sql

```diff
@@ -1,5 +1,20 @@
-/* Database schema to keep the structure of entire database. */
+-- Table: public.animals
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+-- DROP TABLE IF EXISTS public.animals;
+
+CREATE TABLE IF NOT EXISTS public.animals
+(
+    id integer NOT NULL,
+    name text COLLATE pg_catalog."default" NOT NULL,
+    date_of_birth date NOT NULL,
+    escape_attempts integer NOT NULL,
```

Comment:
- [OPTIONAL] You can also add a `species` column with a string type by running this command 👇🏻 instead of dropping the table and recreating it again.
`ALTER TABLE animals ADD species varchar(255);`